zkp/commitments/q_one_way_multiplication.go

```diff
@@ -0,0 +1,206 @@
+package commitmentzkp
+
+import (
+	"github.com/xlab-si/emmy/crypto/common"
+	"math/big"
+)
+
+// ProveCommitmentMultiplication demonstrates how, given commitments A, B, C, prover can
+// prove that C = A * B. Note that commitments need to be based on q-one-way homomorphism
+// (see RSABasedCommitter which is q-one-way homomorphism based).
+func ProveCommitmentMultiplication(homomorphism func(*big.Int) *big.Int, homomorphismInv func(*big.Int) *big.Int,
+	H common.Group, Q, Y,
```

Comment:
Just curious, would it make sense to group any of these parameters (I'm particularly looking at `H, Q, Y, A, B, C`) into a struct with a reasonable name? It would be great if we could reduce the number of parameters.